http/src/test/java/com/redhat/lightblue/client/http/LightblueHttpClientTest.java

```diff
@@ -85,7 +40,7 @@ public void testPojoMappingWithParsingError() throws IOException {
         findRequest.where(withValue("foo = bar"));
         findRequest.select(includeField("_id"));
 
-        String response = "{\"context\": \"rest/FindCommand/esbMessage\", \"errorCode\": \"rest-crud:RestFindError\", \"msg\": \"java.lang.IllegalArgumentException: Cannot call method public static com.redhat.lightblue.crud.FindRequest com.redhat.lightblue.crud.FindRequest.fromJson(com.fasterxml.jackson.databind.node.ObjectNode)\", \"objectType\": \"error\"}";
+        String response = "{\"processed\":\"<p>This is not json</p>\"}";
```

Comment:
Just a note, but this line is actually valid json.  The intent of the test is to have json that violates the json standard.